token.go

```diff
@@ -58,7 +59,7 @@ func (m murmur3Token) Less(token token) bool {
 
 // order preserving partitioner and token
 type orderedPartitioner struct{}
-type orderedToken []byte
+type orderedToken string
```

Comment:
This lets me use orderedToken as a key in the look up map. Required to support token types from different partitioners in a common lookup map (in token ring)

Reply:
it's not necersarry to do this as doing a lookup via `myMap[string(someBytes)]` can be optimised by the compiler to avoid allocations

Reply:
I had to change it to string so that I can use it as a key in a map. If you use a slice of byte, you get an error: hash of unhashable type
Ex: https://play.golang.org/p/EEq2KABDHiV